english/coffee/drink-water/bathroom/bathroom.md

```diff
@@ -0,0 +1 @@
+You quick start running down the street to look for a bathroom. You see a sign for a public toilet 2 mins down the road and you also see a Starbucks.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.